frontend/src/routes/portfolio/+page.svelte

```diff
@@ -1,11 +1,19 @@
+<script>
+    import Activity from "$components/Activity.svelte";
+    import Asset from "$components/Asset.svelte";
+</script>
+
 <div class="block mt-5 ml-5">
     <h1 class="title is-2">Portfolio</h1>
 </div>
 
 <div class="columns mt-5" style="width: calc(100vw + 12px);">
     <div class="column is-two-fifths">
         <h2 class="title is-5 ml-5">Summary</h2>
-        <div class="box mx-5 has-background-grey-darker" style="height: 25vh;">
+        <div
+            class="box mx-5 has-background-grey-darker"
+            style="height: 25vh; overflow-y: scroll;"
```

Comment:
This leaves ugly black bars everywhere (at least on Firefox) - replacing all `overflow-y: scroll;`with `overflow-y: auto` seems to help. Also, I don't think this box needs to be auto-resizable with height: I think making the entire page scrollable (aside from navbar) would work better than compressing all the elements.

Reply:
Would you mind sending a screenshot of what it looks like on your browser to Discord? I can't seem to trigger the black bars on my machine...
I will be replacing the `scroll` with `auto`.
With regards to box auto-resizing: valid point, I'll try to made their height immune to browser resize. I'm less sure about making the entire page scrollable though - maybe talk about it on Tues?

Reply:
Have sent messages in the discord. Also, good job on the portfolio page - resizing works much better now!